tools/handle_rhv_storage_duplication.rb

```diff
@@ -0,0 +1,83 @@
+#!/usr/bin/env ruby
+
+# This script removes duplicate block storages that were created due to changing the way
+# storage "location" is calculated.
+# It copies the tags from the old storages and then removes them from ManageIQ.
+# See https://github.com/ManageIQ/manageiq-providers-ovirt/pull/387
+# And https://bugzilla.redhat.com/show_bug.cgi?id=1697467
+# Recommended usage: run with -dv argument to have a dry run with output.
+# If it makes sense, run with -v
+
+class HandleStorageDuplication
+  attr_reader :ems, :dry_run, :verbose
+  def initialize(opts)
+    @ems = opts[:ext_management_system]
```

Comment:
Only one question: Is there any other usage of the HandleStorageDuplication class? 
@ems will always be nil using HandleStorageDuplication from the script (if I understand well optimist), but maybe could be useful for testing or from other scripts.